File: src/main/Python/boj_step2/boj2480.py
import sys

a, b, c = map(int, sys.stdin.readline().split())
reward = int()
if a == b and a == c: reward = 10_000 + 1_000 * a
elif a == b or a == c: reward = 1_000 + 100 * a
elif b == c: reward = 1_000 + 100 * b
else: reward = max(a, max(b, c)) * 100
print(reward)

File: src/main/Cpp/boj_step2/boj2480.cpp
#include <iostream>
using namespace std;

int main() {
    int a, b, c, reward;
    cin >> a >> b >> c;
    if (a == b && b == c) reward = 10000 + a * 1000;
    else if (a == b || b == c) reward = 1000 + b* 100;
    else if (a == c) reward = 1000 + a * 100;
    else reward = max(a, max(b, c)) * 100;
    cout << reward;
}